src/assign.cpp

```diff
@@ -1355,8 +1355,8 @@ bool fba::Assignment::ok_to_assign (fba::Hardware const * hw, int32_t tile,
                 loc,
                 tpos,
                 nb,
```

Comment:
This is the most important of the theta/phi offset changes.  This is where we're checking whether a neighbor will collide with a stuck/broken positioner.